pandas/core/frame.py

```diff
@@ -11476,35 +11641,137 @@ def interpolate(
             **kwargs,
         )
 
+    @overload
+    def where(
+        self,
+        cond,
+        other=...,
+        *,
+        inplace: Literal[False] = ...,
+        axis=...,
+        level=...,
+        errors: IgnoreRaise | lib.NoDefault = ...,
```

Comment:
`errors` is not used in `NDFrame/Series/DataFrame.where` (and also in `mask`, which just calls `where`). Does `errors` need to go through a depreciation cycle, or can it simply be removed?

Reply:
in either way, I will address it in a follow-up PR

Reply:
It would probably need a deprecation cycle to be safe